unity-client/Assets/Scripts/MainScripts/DCL/Models/DecentralandEntity.cs

```diff
@@ -35,6 +35,27 @@ public GameObject meshRootGameObject
             public MeshFilter[] meshFilters;
             public List<Collider> colliders = new List<Collider>();
 
+            Vector3 lastPosition;
+            Vector3 lastScale;
+            Quaternion lastRotation;
+            Bounds _mergedBounds;
+            public Bounds mergedBounds
+            {
+                get
+                {
+                    if (meshRootGameObject.transform.position != lastPosition ||
```

Comment:
We should only recalculate bounds if the rotation or scale changes. If position changes we just add the position delta to the bounds. This is taking advantage of the fact that most of the scenes just update entities position. Updating rotation and scale is more rare.

If we just check everything, the bounds will be recalculated almost always and will be the same as before.